modules/apps/headless/headless-user-notification/headless-user-notification-test/src/testFunctional/macros/UserNotificationAPI.macro

```diff
@@ -48,26 +48,48 @@ definition {
 	}
 
 	macro createNotificationTemplate {
```

Comment:
great job on this test @davisaints! My only comment is to change this name to postNotificationTemplate since it is an API macro. You may forward after that change.